### PR TITLE
cpu/saml1x: update periph status implementation in board doc.txt

### PR DESCRIPTION
**Pull request by Einhornhool**
### Contribution description
Updated the implementation status of the boards saml10-xpro and saml11-xpro.
Before only support of low-level drivers GPIO, UART and Timer was documented.
Testing showed that I2C, SPI, RTT, RTC, RNG and ADC also work on both boards.

### Testing procedure

I ran the following tests:
periph_i2c
periph_spi
periph_rtt
periph_rtc
periph_adc
rng

### CODE REVIEW

File: boards/saml10-xpro/doc.txt [hunk 1]
```diff
@@ -51,14 +51,14 @@ memory.
...
-|        | SPI       | no        | |
+|        | I2C       | yes        | |
+|        | SPI       | yes        | |
 |        | USB       | no        | |
-|        | RTT       | no        | |
-|        | RTC       | no       |  |
-|        | RNG       | no        |  |
+|        | RTT       | yes        | |
+|        | RTC       | yes      |  |
+|        | RNG       | yes        |  |
```
Comment (benpicco) [inline boards/saml10-xpro/doc.txt]: This still requires #11647 to be merged, but then it would also be true for saml11-xpro.

**Comment by PeterKietzmann** [comment]
I just ran *tests/periph_adc* on a saml11-xpro which works somehow but I think something might be messed up in the peripheral driver implementation. In main.c we sample with 10 bit but the test prints values between ~ 96 ... 65520 for ~0...3.3V which shouldn't be possible @10 bit right? @dylad any ideas?

**Comment by dylad** [comment]
I'll take a closer look at it next week.

**Comment by dylad** [comment]
> @dylad any ideas?

I have one. I'll submit a PR soon so you can fully test it.
Thanks for spotting this @PeterKietzmann !

**Comment by dylad** [comment]
@Einhornhool regarding this PR, do you want to wait for #11647 to be merge first so we can merge this one or you can remove hwrng support from the list and we can merge it right now.

**Comment by Einhornhool** [comment]
@dylad 
I just took a look at that pr. Isn't it already merged? 
I can still remove rng support, if you're not sure it works.

Thank you for working on this =)

**Comment by PeterKietzmann** [comment]
Yeah it's been merged in the meantime, no need to remove RNG support - it's been tested. However, there are open issues with the ADC implementation. @dylad any news on that?

File: boards/saml11-xpro/doc.txt [hunk 2]
```diff
@@ -51,14 +51,14 @@ memory. In addition, this SoC features the ARM TrustZone technology.
...
-|        | I2C       | no        | |
-|        | SPI       | no        | |
+|        | I2C       | yes        | |
+|        | SPI       | yes        | |
 |        | USB       | no        | |
-|        | RTT       | no        | |
-|        | RTC       | no       |  |
+|        | RTT       | yes       | |
+|        | RTC       | yes      |  |
 |        | RNG       | no        |  |
```
Comment (benpicco) [inline boards/saml11-xpro/doc.txt]:
```suggestion
|        | RNG       | yes        |  |
```

**Comment by dylad** [comment]
@PeterKietzmann #11972 should fixes your issue, I have limited access to my hardware because I'm moving so I cannot fully test it right now.

**Comment by PeterKietzmann** [comment]
ADC's are fine with #11972. @Einhornhool please adopt @benpicco's comment. We are ready to merge afterwards.

**Comment by Einhornhool** [comment]
@PeterKietzmann Thanks, will do. Guess I made a mistake there.

**Comment by benpicco** [comment]
I'd say you can squash right away.

**Comment by Einhornhool** [comment]
@benpicco done

**Pull request merged**